pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/defaultvalues/ATDefaultValues.java

```diff
@@ -42,9 +42,11 @@ public class ATDefaultValues extends BasicDefaultDataMaker {
     private Dao dataAccess;
 
     @Override
-    protected boolean setDefaultData() {
+    boolean setDefaultData() {
         if (isParticipantDataNotInES()) {
-            throw new ESMissingParticipantDataException("Participant does not yet have profile and activities in ES");
+            throw new ESMissingParticipantDataException(
+                    String.format("Participant %s does not yet have profile and activities in ES",
+                    elasticSearchParticipantDto.getQueriedParticipantId()));
```

Comment:
I'm not sure I understand when the queried ptp ID would ever be set in this case.

Reply:
It'll be set in `ElasticSearch.parseSourceMap`: https://github.com/broadinstitute/ddp-study-server/pull/2722/files#diff-9b912998c53490bce5548304c2dbaa2977c084100ace3d6b8d38bceafcd1e8ed